File: code/LinearSearch.java
public class LinearSearch {
    public static void main(String[] args){
        int[] numbers = new int[20];
        for(int i = 0; i < 20; i+=1) numbers[i] = 1 + i*2;

        System.out.println(LinearSearch.search(numbers, 11));
    }

    public static int search(int[] numbers, int target){
        int lo = 0;
        int hi = numbers.length - 1;
        
        for(int i = lo; i < hi; i+=1){
            if(numbers[i] == target) return i;
        }

        return -1;
    }
}

File: code/linear_search.py
# Assuming we have a list of numbers 1,3,5,7,9,...
# and the question is, what is the index of number '11'
listOfNumbers = [i*2+1 for i in range(20)]
toFind = 11
# search from the first index to the last index
for i in range(0, len(listOfNumbers)):
    if listOfNumbers[i] == toFind:
        print("Found", toFind, "at index", i)
        break
